### ISSUE TITLE
Gather mean, variance information for given model

### ISSUE THREAD

**Issue by jvia** [jvia/fyp#12]
Run the model against the observation used to generate it in order to determine the mean and the variance of the score.

**Comment by hawesie** [comment]
Wow, I never knew this was here.

**Comment by jvia** [comment]
It's nice to write down goals. My planning is split between here, my org file, and this gmail add-on: http://www.streak.com

I'm still undecided about which I like best.

**Comment by jvia** [comment]
The system now runs its trained model against the observation input as the trained model and then score mean and variance are calculated.
- I set the threshold as 1/2 the mean because this has worked well in recent experiments. Originally, I set the threshold to the mean minus two standard deviations but because the variance is calculated by running the model against its own training data, the variance is minuscule.
- Also, fixed what I suspect was an error in Raph's implementation. In the score calculation in his paper, the total entropy of the distributions related to the input are used to modify the score, but in his implementation he squared this entropy which meant that when it came time to calculate the inverse entropy, it had no effect on the score.
